src/index.html

```diff
@@ -0,0 +1,10 @@
+<!DOCTYPE html>
+<html lang="ru">
+  <head>
+    <meta charset="UTF-8">
+    <title>Hello World</title>
+  </head>
+  <body>
+    <h1>Hello World</h1>
```

Comment:
Remove `h1` tag. Task said:

Please do not use any enclosing tags like `<h1>` or `<p>`.

Reply:
fixed everything, I just needed to update